### DISCUSSION TITLE
is it possible to use 'net' node library in electron-preload.js

### DISCUSSION THREAD

**Discussion by on-meetsys** [quasarframework/quasar#14225] [CLI - Electron mode]
Hello,

I have to use 'net' services in a p2p Quasar/Electron application, for example : 
```
import net from 'net'
const server = net.createServer();
```
It works fine in `electron-main.js` file : the server is created.

I would also like to call methods from the renderer, so I've tried to do it in the `electron-preload.js` file. But then, the 'net' library seems to be imported thanks to webpack, and with the same code I have the following error : 
`p2p.ts?1f3e:13 Uncaught (in promise) TypeError: net__WEBPACK_IMPORTED_MODULE_7___default(...).createServer is not a function`
Is there a Quasar setup to have Webpack not used in `electron-preload.js` ? Or do I have to put my 'net' code into  `electron-main.js` and call them by messages ? Or do I have misunderstood something ?

Thanks for any advice.

**Comment by tuckerww** [comment]
FYI for others finding this page in the future:  It might be including the Electron `net` instead of the Node `net`.  The former doesn't appear to offer the `createServer()` method.
